### PR TITLE
Depend on http://github.com/chrismattmann/tika-python from pip (=1.8.7)

### PR DESCRIPTION
**Pull request by chrismattmann**

### CODE REVIEW

**Comment by brittainhard** [comment]
@chrismattmann I'm confused. Why was tika.py deleted?

**Comment by chrismattmann** [comment]
Bc it's available and originally from the tika-Python library now available in pip. I believe it was originally copied from there into memex-explorer. This dependency adds it in via pip.

**Comment by brittainhard** [comment]
Ahh, I see. Thanks for the clarification.

**Comment by chrismattmann** [comment]
Did i mess it up @brittainhard not sure why the build failed

File: environment.yml [hunk 1]
```diff
@@ -17,6 +17,7 @@ dependencies:
     - jinja2
     - redis
     - redis-py
+    - tika=1.8.7
```
Comment (ahmadia) [inline environment.yml]: @chrismattmann - Did you test this locally?  This tries to use the tika 1.8.7 package from binstar, which I'm guessing hasn't been uploaded yet.

File: environment.yml [hunk 2]
```diff
@@ -17,6 +17,7 @@ dependencies:
...
     - pip:
         - tld
         - selenium
```
Comment (ahmadia) [inline environment.yml]: @chrismattmann - This is where we'd put a `pip install tika=1.8.7` command.  Is this what you intended?

**Comment by ahmadia** [comment]
@chrismattmann - Yes, I copied it in.  I think your approach to replacing our hard-coded version with the PyPI installation from your repository looks correct, we just need to move it to the pip install section of the environment file.

**Comment by brittainhard** [comment]
Yeah @chrismattmann there isnt a conda tika package, so you have to add the packages under the pip: section, then the build should work.

**Comment by chrismattmann** [comment]
Please see #484

**Pull request closed without merging**